src/variables/components/VariableForm.tsx

```diff
@@ -115,6 +115,7 @@ export default class VariableForm extends PureComponent<Props, State> {
             <Grid.Row>
               <Grid.Column>
                 <VariableArgumentsEditor
+                  variables={[]}
```

Comment:
jumping in with the clarity of a half context afforded by PR reviewing. is this case true? variables can be queries that reference queries.

Reply:
This is the form to input a new variable (which is a query). 
* It uses the flux editor, and therefore needs a variables array. 
* But since it's only creating a new variable, and not consuming existing variables (a.k.a. no variable injection into the flux editor) -- it only needs an empty array.

<img width="500" alt="Screen Shot 2022-05-17 at 10 52 50 AM" src="https://user-images.githubusercontent.com/10232835/168878696-33a0beab-b855-40fa-b792-0d8b098c8a25.png">